ert_storage/database_schema.py

```diff
@@ -24,6 +24,20 @@ class RecordClass(Enum):
     other = 3
 
 
+class PriorFunction(Enum):
```

Comment:
I think its time to start splitting this file up. starting by moving the enums in to a separate file